docs/source/demos/text_input.ipynb

```diff
@@ -237,6 +237,94 @@
     "As you can see, this model performs relatively well on this dataset, even on unseen data."
    ]
   },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "## What does the Text Featurization Component do?\n",
+    "\n",
+    "Machine learning models cannot handle non-numeric data. Any text must be broken down into numeric features that provide useful information about that text. The Text Featurization Component starts by normalizing your text: it removes any punctuation and other non-alphanumeric characters, and converts any capital letters to lowercase. From there, it passes the text into featuretools' `nlp_primitives` dfs search, resulting in several informative features that replace the original column in your dataset: Diversity Score, Mean Characters per Word, Polarity Score, and LSA (Latent Semantic Analysis).\n",
+    "\n",
+    "**Diversity Score** is the ratio of unique words to total words\n",
+    "\n",
+    "**Mean Characters per Word** is the average number of letters in each word\n",
+    "\n",
+    "**Polarity Score** is a prediction of how \"polarized\" the text is, on a scale from -1 (extremely negative) to 1 (extremely positive)\n",
+    "\n",
+    "**Latent Semantic Analysis** is an abstract representation of how important each word is with respect to the entire text, reduced down into two values per text. While the other text features are each a single column, this feature adds two columns to your data, `LSA(column_name)[0]` and `LSA(column_name)[1]`.\n",
+    "\n",
+    "Let's see what this looks like with our spam/ham example."
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "best_pipeline.input_feature_names"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "Here, the Text Featurization component takes in a single \"Message\" column, but then the next component in the pipeline, the Imputer, recieves five columns of input, the result of featurizing the text-type \"Message\" column. Most importantly, these featurized columns are what ends up passed in to the estimator.\n",
```

Comment:
recieves --> receives

Just a suggestion but maybe split this up into something like:

Here, the Text Featurization component takes in a single \"Message\" column, but then the next component in the pipeline, the Imputer, receives five columns of input. These five columns are the result of featurizing the text-type \"Message\" column.